src/dali_model_instance.h

```diff
@@ -62,7 +62,8 @@ class DaliModelInstance : public ::triton::backend::BackendModelInstance {
     auto serialized_pipeline = dali_model_->GetModelProvider().GetModel();
     auto max_batch_size = dali_model_->MaxBatchSize();
     auto num_threads = dali_model_->GetModelParamters().GetNumThreads();
-    DaliPipeline pipeline(serialized_pipeline, max_batch_size, num_threads, GetDaliDeviceId());
+    DaliPipeline pipeline(serialized_pipeline, max_batch_size, num_threads, GetDaliDeviceId(),
```

Comment:
What would happen if we load the same plugin .so twice?

Reply:
Nothing. It would likely break, though, if there were two copies of the library loaded from different paths - both would be initialized, but the operator name would clash resulting in weird behavior. If the maps are consistently populated with `insert`, the one loaded first would prevail; if they are populated with `[]` and assignment - the one loaded last would prevail. If there's a mix of the two assignment techniques, there would be a mess, leading to all sorts of weird bugs.